lib/Date/Holidays/Adapter/BY.pm

```diff
@@ -0,0 +1,121 @@
+package Date::Holidays::Adapter::BY;
+
+use strict;
+use warnings;
+use vars qw($VERSION);
+
+use base 'Date::Holidays::Adapter';
+
+$VERSION = '1.03';
```

Comment:
This version number has to reflect the distribution version number so package/distribution relation is clean, so it should read 1.04